### ISSUE TITLE
Request for Assistance with Installation Process

### ISSUE THREAD

**Issue by MRX-8** [ramykatour/cryptitan#1]
installation issue

**Comment by ramykatour** [comment]
SERVER SPECIFICATION
OS: Linux (Ubuntu 18.04 or later)

Architecture: 64bit (x86)

AVX support on CPU

Memory: 8gb+

Storage: 60gb+

**Comment by MRX-8** [comment]
Yes bro I'm aware. Please help my life. Help me out on how I'll install it. I followed the documentation link of the project but it's confusing. I've tried but it's not working. Please I'll pay. If necessary. I'm begging you
